### ISSUE TITLE
terraform version not showing plugin version

### ISSUE THREAD

**Issue by pdion891** [cloud-ca/terraform-provider-cloudca#44]
The command `terraform version` does not display the cloud.ca driver version. 

```
$ terraform version

Terraform v0.11.1
+ provider.cloudca (unversioned)
+ provider.template v1.0.0

```

We should also update our installation documentation to state the plugin must now be copied into `~/.terraform.d/plugins/darwin_amd64`

**Comment by pdion891** [comment]
Look like `mv ~/.terraform.d/plugins/terraform-provider-cloudca ~/.terraform.d/plugins/terraform-provider-cloudca_v1.0.2` does the trick.

**Comment by pdube** [comment]
We will have to update the Makefile to include the version in the binary name. Thanks for reporting the issue

**Comment by pdube** [comment]
Fixed in version v1.1.0
